### ISSUE TITLE
Strong tag not working when in an ordered list and the list number is multiple digits

### ISSUE THREAD

**Issue by davidmeekstockport** [lunet-io/markdig#134]
Hi Mark Dig,

I seem to be having an issue when converting some markdown to html. The markdown is as follows:

`11.	 __test__ hello`

It seems to convert to this:

```
<li>
    <pre>
        <code>est__ hello</code>
    </pre>
</li>
```

I'm not entirely sure how this is happening, or if there are any extensions I can use to bypass this issue. This issue appears on the latest version 0.13.0. Any ideas?

Thanks,

David

**Comment by xoofx** [comment]
Can't reproduce this issue on [babelmark](https://babelmark.github.io/?text=11.+__test__+hello) 

It seems that you have this issue in a context not described here...

**Comment by davidmeekstockport** [comment]
I've done some further investigation into this issue and have found the following.

When executing this code:

`var html = Markdown.ToHtml("10.\t*test* – test\n\n11.\t__test__ test\n\n", new MarkdownPipelineBuilder().Build());`

I get back 

```
<ol start="10">
    <li>
        <pre>
            <code>t* – test</code>
        </pre>
    </li>
    <li>
        <pre>
            <code>st__ test</code>
        </pre>
    </li>
</ol>
```

However if I were to take out the `\t` I get this:

var html = Markdown.ToHtml("10. *test* – test\n\n11. __test__ test\n\n", new MarkdownPipelineBuilder().Build());

<ol start="10">
    <li>
        <p><em>test</em> – test</p>
    </li>
    <li>
        <p><strong>test</strong> test</p>
    </li>
</ol>

And again this only seems happens if the list numbers are double digits. I also can't recreate this on babelmark.

**Comment by xoofx** [comment]
Looks like a bug in markdig on [babelmark](https://babelmark.github.io/?text=10.%09*test*+%E2%80%93+test%0A11.%09__test__+test)... will have to check this...

**Comment by xoofx** [comment]
This should be fixed by commit bb30dc2 and available in 0.13.1
